### ISSUE TITLE
PHP 7.2 each() function in CTabView is deprecated

### ISSUE THREAD

**Issue by phena109** [yiisoft/yii#4199]
As stated in http://php.net/manual/en/function.each.php and http://php.net/manual/en/migration72.deprecated.php#migration72.deprecated.each-function that the each() function is deprecated in PHP 7.2

There are 2 other instances of each() in `/framework/gii/components/Pear/Text/Diff/Engine/native.php` if the fix could extend a little bit further.

### What steps will reproduce the problem?
Simply render any CTabView widgets (precisely calling the run() function)

e.g. `$this->widget('CTabView', $config);`

### What is the expected result?
The run() function should run without throwing any PHP error

### What do you get instead?
A PHP Error is triggered when using the each() function in PHP 7.2:

`The each() function is deprecated. This message will be suppressed on further calls`

in `/yii/framework/web/widgets/CTabView.php(134)`

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 1.1.20-dev
| PHP version      | 7.2
| Operating system | Mac 10.13.4 (dev) and Ubuntu 16.04.4 (production)

**Comment by abramovk** [comment]
when there will be release 1.1.20?

**Comment by samdark** [comment]
When [all 7.2 issues](https://github.com/yiisoft/yii/issues) will be fixed.

**Comment by abramovk** [comment]
Thank you. We are all waiting for the release!

**Comment by samdark** [comment]
@aztechowski would you please pull-request your changes?

**Comment by surgat** [comment]
Now the first tab is not selected by default

```
-			list($this->activeTab, )=each($this->tabs);
+			$this->activeTab = current($this->tabs);
```

each return array [key, value] 
current return value

activeTab check
```php
protected function renderBody()
{
	foreach($this->tabs as $id=>$tab)
	{
                 $inactive=$id!==$this->activeTab?' style="display:none"' : '';
                 echo "<div class=\"view\" id=\"{$id}\"{$inactive}>\n";
```

Fixed in #4211

**Comment by samdark** [comment]
Merged it.
